client/data/marketplace/use-marketplace-reviews.ts

```diff
@@ -133,15 +144,15 @@ const deleteReview = ( {
 };
 
 export const useMarketplaceReviewsQuery = (
-	{ productType, slug }: ProductProps,
+	{ productType, slug, page, perPage }: MarketplaceReviewsQueryProps,
 	{
 		enabled = true,
 		staleTime = BASE_STALE_TIME,
 		refetchOnMount = true,
 	}: MarketplaceReviewsQueryOptions = {}
 ) => {
-	const queryKey: QueryKey = [ queryKeyBase, slug ];
-	const queryFn = () => fetchMarketplaceReviews( productType, slug );
+	const queryKey: QueryKey = [ queryKeyBase, productType, slug, page, perPage ];
```

Comment:
This is needed because every page should be considered a different query

Reply:
Thanks for adding that @WBerredo!